daemons/execd/execd_commands.c

```diff
@@ -917,8 +917,8 @@ action_complete(svc_action_t * action)
                     }
                 }
             }
-        } else if (pcmk__str_any_of(cmd->action, PCMK_ACTION_MONITOR, PCMK_ACTION_STATUS, NULL) && 
-                  (cmd->interval_ms > 0)) { 
+        } else if (pcmk__str_any_of(cmd->action, PCMK_ACTION_MONITOR, PCMK_ACTION_STATUS, NULL) &&
```

Comment:
same comment as https://github.com/ClusterLabs/pacemaker/pull/3805/commits/a497f6a45222b9949842b8b3585fe0f7c31d1e76#r1938248123 re: `&&` but whatever, it's preference and was never added to the style guide

With that said, I care more about stylistic consistency than about any particular stylistic preference. As another example, I prefer the opening brace on a separate line for function definitions (which happens to be in our C style guidelines), while your preference seems to be same-line. Ultimately, I don't care, as long as we're consistent about which one we use.

For me, a consistent style helps a lot with the cognitive load required to read and understand the code

Reply:
We have indent support in `devel/Makefile.am`, but indent is kind of old at this point and I know people tend to have various complaints about it.  There's also clang-format.  Perhaps it'd be worth spending some time configuring that and then making sure everyone's using it in their editors (or we could do a commit hook, github action, etc.)